system-addon/content-src/components/TopSites/TopSitesEdit.jsx

```diff
@@ -121,8 +133,8 @@ export class _TopSitesEdit extends React.PureComponent {
           <div className="modal-overlay" onClick={this.onModalOverlayClick} />
           <div className="modal">
             <TopSiteForm
-              label={this.props.TopSites.rows[editIndex].label || this.props.TopSites.rows[editIndex].hostname}
-              url={this.props.TopSites.rows[editIndex].url}
```

Comment:
Not sure if this is any cleaner, but we could avoid the added `editUrl` variables by just getting it inline: `url={(this.props.TopSites.rows[editIndex] || {}).url}` ?